src/Common/perf/PerfRunner/project.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0.0-*",
```

Comment:
I don't have a strong preference either way I just know that it can sometimes be more difficult to make changes to things that live in buildtools as compared to CoreFX.  However, I do not believe that we will be needing to make a bunch of changes to this file so that might not be an issue.